crates/consensus/beacon/src/engine/invalid_headers.rs

```diff
@@ -82,7 +83,7 @@ impl<H: Debug> InvalidHeaderCache<H> {
 struct HeaderEntry<H> {
```

Comment:
I think this should just be 
```rust
struct HeaderEntry {
    /// Keeps track how many times this header has been hit.
    hit_count: u8,
    /// The actual header entry
    header: BlockWithParent,
}
```

generic is not useful anymore

Reply:
whoops, yep